integration/fuse/src/test/java/alluxio/fuse/AlluxioFuseFileSystemTest.java

```diff
@@ -492,4 +492,9 @@ private FileInfo setUpOpenMock(AlluxioURI uri) throws Exception {
     when(mFileSystem.getStatus(uri)).thenReturn(status);
     return fi;
   }
+
+  @Test
+  public void statfs() throws Exception {
+    // TODO(LuQQiu): implement this unit test.
```

Comment:
@liuhongtong Could you change to `TODO(lu)`, thanks!

Reply:
I pushed a change to address this.